LocalSettings.php

```diff
@@ -1793,6 +1793,12 @@
 		],
 	],
 
+	// InterwikiSorting
+	'wgInterwikiSortingSort' => [
+		'default' => 'code',
+		'gratispaideiawiki' => 'alphabetic',
```

Comment:
We don't support this yet actually. This is not built in to the extension and only supported by the WMF.

Reply:
> We don't support this yet actually. This is not built in to the extension and only supported by the WMF.

Did you add it to ManageWiki?